explorer/src/main/java/com/scalar/client/tool/explorer/command/Validate.java

```diff
@@ -51,10 +52,18 @@
   public void run() {
     ExplorerExecutor executor =
         explorer -> {
+          StatusCode statusCode;
           try {
             for (String assetId : assetIds) {
-              explorer.validate(assetId);
-              System.out.println(assetId + " is not tampered");
+              statusCode = explorer.validate(assetId).getCode();
+
+              if(statusCode.equals(StatusCode.OK)) {
```

Comment:
Format please  !

Reply:
Thank you for review comments.
Sorry, I will check before the push
Updated review changes in fafec79